src/modules/Dashboard/features/Applet/Activities/Activities.test.tsx

```diff
@@ -166,12 +166,9 @@ describe('Dashboard > Applet > Activities screen', () => {
         routePath,
       });
 
-      await waitFor(() =>
-        expect(screen.getAllByTestId(`${testId}-activity-actions-dots`)[0]).toBeVisible(),
-      );
-      userEvent.click(screen.getAllByTestId(`${testId}-activity-actions-dots`)[0]);
-
-      if (canEdit) {
+      const actionDots = screen.queryAllByTestId(`${testId}-activity-actions-dots`)[0];
+      if (actionDots && canEdit) {
+        userEvent.click(screen.getAllByTestId(`${testId}-activity-actions-dots`)[0]);
```

Comment:
Can you use the existing reference to `actionDots`?

```suggestion
        userEvent.click(actionDots);
```